src/concepts/gaining_access.md

```diff
@@ -0,0 +1,14 @@
+Access to Firefox's telemetry data is limited to two groups:
+
+1. Mozilla employees and contractors
+2. Contributors who have signed an [NDA](https://wiki.mozilla.org/NDA),
```

Comment:
```suggestion
2. Contributors who have signed a [non-disclosure agreement](https://wiki.mozilla.org/NDA),
```

Reply:
This would allow removing `NDA` from the spelling list

Reply:
Great to remove jargon where we can! Thanks for the suggestion